main.py

```diff
@@ -33,6 +50,10 @@
     data_ampl.append( tmp_ampl[pos] ) 
     data_freq.append( tmp_freq[pos] )
     data_db.append( 10.0*n.log10(n.abs(tmp_ampl[pos])) )
```

Comment:
There is something to look at in L43 - L45

```suggestion
    print('hello')
```

> 44

Reply:
https://github.com/Naireb/live_freq_analysis/blob/naireb/main.py#L65-L74